src/App.tsx

```diff
@@ -1,14 +1,56 @@
-import React from 'react';
+import React, {
+  useState,
+  useCallback,
+  useMemo,
+} from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { List } from './components/PeopleDropDownList';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const copyPeople = [...peopleFromServer];
+  const [query, setQuery] = useState('');
+  const [focused, setFocused] = useState(false);
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [delayedQuery, setQuerySearch] = useState('');
```

Comment:
```suggestion
  const [delayedQuery, setDelayedQuery] = useState('');
```
always keep state names like this